src/models/metadata.rs

```diff
@@ -318,10 +318,7 @@ mod tests {
         let owner_private_key = PrivateKey::from_ed25519(OWNER_PRIVATE_KEY).unwrap();
         let layout_metadata = Box::new(
             LayoutMetadataBuilder::new()
-                .expires(DateTime::<Utc>::from_utc(
-                    NaiveDateTime::from_timestamp(0, 0),
-                    Utc,
-                ))
+                .expires(Utc.from_utc_datetime(&NaiveDateTime::from_timestamp_opt(0, 0).unwrap()))
```

Comment:
can we wrap this better so it fits 80 chars?

Reply:
Looks like rustfmt wrap limit is 100, not 80. After this is merged, look at #73

Reply:
ugh, this is going to be noisy, thanks!